### ISSUE TITLE
ChatSecure iOS isn‘t working

### ISSUE THREAD

**Issue by acybl90** [ChatSecure/ChatSecure-iOS#1125]
Hello,
I have the following problem, the app is currently not usable, because it can not connect to my XMPP Server and shows me instead error messages all the time. Can anyone help?
The app has always worked well and now this...
![846738B0-84C0-4344-9B3D-8C67F61D1A82](https://user-images.githubusercontent.com/51682942/59236482-a4866e80-8bf6-11e9-8c38-61c137dd2665.jpeg)

**Comment by acybl90** [comment]
push

**Comment by mwlczk** [comment]
Can you connect with another client? Is it your own server or a public one, if public which one?

**Comment by acybl90** [comment]
I can log in in my account, but it shows me these error messages. I use a public server, @xmpp.jp

**Comment by licaon-kter** [comment]
@acybl90 that server looks ok ( https://compliance.conversations.im/server/xmpp.jp/ )

Can you test with another server?

**Comment by acybl90** [comment]
Yes i have tested with@exploit.im, also doesnt work, i think it has nothing to do with the server, but with the app

**Comment by licaon-kter** [comment]
@acybl90 Of all those servers [at 100%](https://compliance.conversations.im/) you tested with one at 31% ( https://compliance.conversations.im/server/exploit.im/ ) ?

Make an account on trashserver.net or creep.im or draugr.de and retest.

**Comment by acybl90** [comment]
I tested with trashserver and it also didnt worked. Does your app work?
